docs/imaging/dlc.rst

```diff
@@ -0,0 +1,114 @@
+.. _Imaging dlc:
+
+=========================
+Imaging: DLC
+=========================
+
+`DeepLabCut <http://www.mackenziemathislab.org/deeplabcut>`_ is a software package for inferring motion capture information from tracking video. Whereas Optitrack makes use of one infra-red reflectors attached to the subject(s), DeepLabCut instead uses a deep neural network trained on human-labelled data of the subject(s), allowing for less interference with subject behaviour. 
```

Comment:
`Whereas Optitrack makes use of infra-red reflectors that are physically attached to the body parts that have to be tracked,  DLC ... `